docs/hello_world.md

```diff
@@ -0,0 +1,11 @@
+# Hello World!
+
+After you have learnt how to use the executable go to the directory that you want to make your first program in, create a file with a `.n` extension (preferrably `run.n` but it can be a different name) then add the following code there
```

Comment:
```suggestion
After you have learnt how to use the executable go to the directory that you want to make your first program in, create a file with a `.n` extension (preferably `run.n` but it can be a different name) then add the following code there
```

https://www.grammarly.com/blog/preferably-preferrably/